packages/react-composites/src/composites/CallWithChatComposite/adapter/CallWithChatAdapter.ts

```diff
@@ -276,6 +280,12 @@ export interface CallWithChatAdapterManagement {
   /* @conditional-compile-remove(file-sharing) */
   /** @beta */
   updateFileUploadMetadata: (id: string, metadata: FileMetadata) => void;
+  /* @conditional-compile-remove(PSTN-calls) */
+  /** @beta */
+  toggleHold: () => Promise<void>;
+  /* @conditional-compile-remove(PSTN-calls) */
+  /** @beta */
```

Comment:
nit: these should have a doc comments with some explaination (although pretty obvious)